src/main/java/com/programmerdan/minecraft/simpleadminhacks/SimpleAdminHacks.java

```diff
@@ -46,9 +45,6 @@ public void onEnable() {
 	@Override
 	public void onDisable() {
 		this.commands.reset();
-		if (Bukkit.getPluginManager().isPluginEnabled("ProtocolLib")) {
-			ProtocolLibrary.getProtocolManager().removePacketListeners(this);
-		}
```

Comment:
Just for future reference, this would rarely, if ever work. It's clear this is a soft-dependency check, but the import would immediately cause an exception because if ProtocolLib is not loaded, how is it going to import ProtocolLib classes? Doesn't matter if the classes aren't being used until you know ProtocolLib is enabled. This is coincidentally why soft-dependencies are so finicky.

Reply:
Actually this should be fine based on when I've done this before

Reply:
Huh, correction noted. Just tested and you are correct: imports do not initialise the class. Not sure whether this is a new feature since I had a rather frustrating time trying to get optional dependencies to cooperate nicely a few months ago where any reference to unloaded classes meant exceptions. Either way though, that call to remove all the plugin's packet listeners is a nuclear solution to the problem of hacks not cleaning up after themselves, which has been solved by the new `PacketManager` class.

Reply:
You probably had the dependency as a class member in a class that you were initializing, which would cause it to attempt to classload it. I've had the same.